ui/src/widgets/ui-text-input/UITextInput.vue

```diff
@@ -1,19 +1,19 @@
 <template>
     <v-tooltip :text="tooltip" :disabled="!tooltip?.length" location="bottom">
         <!-- eslint-disable-next-line vue/no-template-shadow -->
-        <template #activator="{ props }">
+        <template #activator="{ props: activatorProps }">
```

Comment:
`activatorProps` isn't a variable here?

Reply:
I've to set an alias for the `template activator's props` because I'm using `props.height` coming from the parent component to set the rows and it's getting override by the `template activator's props` 
<img width="816" alt="Screenshot 2024-09-24 at 15 54 57" src="https://github.com/user-attachments/assets/46534e1e-94a1-4158-9c36-f4795ef80d43">